action.yml

```diff
@@ -3,8 +3,16 @@ description: 'Exports .env file variables to job env'
 author: 'cardinalby'
 inputs:
   envFile:
-    description: 'Path to .env file'
+    description: 'Path to .env file or Path to multiple .env files concatenated by `|` or custom "separator" prop'
     required: true
+  variables:
```

Comment:
Good questions. I do not want to expose to the action all the environment variables...I just need some of them, and I never know if there are new environment variables added, so excluded is not enough, because I want to retrieve only the once that I need.
So, to answer each point:

1. I want to filter in both scenarios...it doesn't matter if it's action inputs or exported variables, it makes sense to specify and avoid mistakes or wrong usage of another variable...so it's a global definition.
2. That is the tricky one...but I think it makes sense to participate on extending variables, because if you need a variable that extends from other one, you should get the right value...it doesn't matter if you don't need in the output the intermediate variable.
3. Yes, I think everything should work as it was working from before...the only thing that changes is the final output/export...but everything inside the process should work as it was working
4. From my perspective it doesn't make sense. I worked with secrets and environment variables from a security perspective many times...and the idea is always to only expose what you need, nothing else. And if you exclude some variables is not the same behaviour...because any new addition to the .env file, you have to keep updated the excluded definition, when probably that new variable is not associated neither required in the action.
5. I added unit tests to cover different workflows related to variables with extend, mask, etc...from before. I think I covered every case.
6. And I also updated the documentation and action.yml with this definition. You can find input definition in the README file for the variables, and also examples with variables

Reply:
Agree on all points, just wanted to ensure you understand the desired behaviour and to be explicit about it, but this behaviour should be documented